_usage/query_planning.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: default
+title: Reviewing a Query Plan
+nav_order: 8
+---
+
+## Reviewing a Query Plan
+
+Babelfish version 2.1.0 includes support for functions and parameters that allow you to review estimated and actual (executed) query plans. This allows you to use the TDS port to identify and refine slow-performing queries. The functions can display an query plan for:
```

Comment:
what does 'support for functions and parameters' mean? this is nto clear

Reply:
changed to 'SET statements'

Reply:
changed to 'SET statements'